### PR TITLE
Add flow decomposition documentation

### PR DESCRIPTION
**Pull request by murgeyseb**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Adds the documentation for the new flow decomposition feature developed in powsybl-entsoe repository.

### CODE REVIEW

File: pages/documentation/index.html [hunk 1]
```diff
@@ -397,6 +397,20 @@ <h5>
...
+    <!-- Flow decomposition -->
+    <div class="col-lg-4 col-sm-6 my-2">
+        <div class="media documentation_item">
+            <div class="media-body">
+                <h5>
+                    <a class="documentation_title" href="simulation/flowdecomposition">
+                        <em class="doc-icon align-self-start bx bxs-pie-chart-alt-2"></em>Flow decomposition
+                    </a>
+                </h5>
+                <div class="documentation_descr">Computes for each network element a decomposition of the active flow into different parts (loop flows, allocated flows,...)</div>
```
Comment (annetill) [inline pages/documentation/index.html:410]: Compute to be coherent with the other bullets.

File: pages/documentation/simulation/flowdecomposition/index.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+The aim of flow decomposition algorithm is to provide for each network element a decomposition of the active flow into different parts:
+- Allocated flow: flow due to electricity market exchanges. This includes import/export flows and transit flows.
+- Internal flow: flow due to electricity exchange inside the network element's zone.
+- Loop flow: flow due to electricity exchange inside another zone.
+- PST flow: flow due to a shift commanded by the action of an active phase shifting transformer on the network.
+- HVDC flow: flow due to a shift commanded by the action of an active HVDC line on the network.
+
+> Current algorithm does not model HVDC flow yet.
+
+> This decomposition does not reflects the exact reality of how electric flows act on a real network, but it
```
Comment (annetill) [inline pages/documentation/simulation/flowdecomposition/index.md:28]: reflects -> reflect ?

File: pages/documentation/simulation/flowdecomposition/index.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+> is a useful approximation needed for some cross zonal coordination processes.
+
+## Algorithm description
+
+The flow decomposition algorithm is based on the DC approximation, in which the losses in the network branches
+are neglected, and that allows to rely on the superposition principle to assess which is the impact of any injection
+on any branch flow by simple sensitivity analysis.
```
Comment (annetill) [inline pages/documentation/simulation/flowdecomposition/index.md:35]: on -> or ?

File: pages/documentation/simulation/flowdecomposition/index.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+![Losses compensation on lines](./img/lossesCompensationOnLine.svg)
+
+A special treatment is done on tie lines, where instead of compensating the losses on the sending terminal, losses are
+compensated at both sides proportionally to the resistance of each half line.
+
+![Losses compensation on tie lines](./img/lossesCompensationOnTieLine.svg)
+
+### Nodal Injections partitioning
+
+In order to distinguish internal/loopflows and allocated flows, the nodal injections in each zone must de decomposed in two parts:
```
Comment (annetill) [inline pages/documentation/simulation/flowdecomposition/index.md:67]: loopflows -> loop flows?

File: pages/documentation/simulation/flowdecomposition/index.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+- Nodal injections for allocated flows
+- Nodal injections for loop flows and internal flows
+
+This decomposition is based on GLSK (Generation and Load Shift Keys). It is an input of the process that provides,
+for each zone of the study a list of injections and associated factor to be used to scale the zone to a given net position.
+
+By default, the algorithm uses so-called "Country GSK", which is an automatic GLSK that scales on all generators
+proportionally to their target power set-point.
```
Comment (annetill) [inline pages/documentation/simulation/flowdecomposition/index.md:75]: setpoint

File: pages/documentation/simulation/flowdecomposition/index.md [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+\mathrm{F}_\mathrm{AF} = \mathrm{PTDF} \cdot \mathrm{NI}_\mathrm{AF} \\
+\mathrm{F}_\mathrm{LIF} = \mathrm{PTDF} \cdot \mathrm{diag}(\mathrm{NI}_\mathrm{LIF}) \cdot \mathrm{AM} \\
+\mathrm{F}_\mathrm{PST} = \mathrm{PSDF} \cdot \mathrm{\Delta}_\mathrm{PST}
+\end{array}
+$$
+
+where:
+- $$\mathrm{F}_\mathrm{AF}$$ is the vector of the network element allocated flow, 
+- $$\mathrm{F}_\mathrm{LIF}$$ is the matrix of the network element loop flow or internal flow for each zone, 
+- $$\mathrm{F}_\mathrm{PST}$$ is the vector of the network element PST flow,
```
Comment (annetill) [inline pages/documentation/simulation/flowdecomposition/index.md:117]: You have to explicite somewhere what is PST.

File: pages/documentation/simulation/flowdecomposition/index.md [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+|-----------------------------------------|---------|---------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| save-intermediates                      | boolean | false                                       | When set to true, intermediate results (PTDF matrix, PSDF matrix, etc) are saved in results object. Otherwise only keep flow parts for each XNECs.                                                                                                                                                                          |
```
Comment (annetill) [inline pages/documentation/simulation/flowdecomposition/index.md:173]: Can you explicit XNECs ?

File: pages/documentation/simulation/flowdecomposition/index.md [hunk 8]
```diff
@@ -0,0 +1,183 @@
...
+| xnec-selection-strategy                 | enum    | XnecSelectionStrategy.ONLY_INTERCONNECTIONS | When set to ```XnecSelectionStrategy.ONLY_INTERCONNECTIONS```, the branches are only selected using the interconnection rule. When set to ```XnecSelectionStrategy.ZONE_TO_ZONE_PTDF_CRITERIA```, zonal PTDF are computed and used to select additional branches with 5% zone to zone rule.                                 |
+| dc-fallback-enabled-after-ac-divergence | boolean | true                                        | Defines the fallback bahavior after an AC divergence Use True to run DC loadflow if an AC loadflow diverges (default). Use False to throw an exception if an AC loadflow diverges.                                                                                                                                          |
```
Comment (annetill) [inline pages/documentation/simulation/flowdecomposition/index.md:179]: Bahavoir ?

File: pages/documentation/simulation/flowdecomposition/index.md (hunk 3 above)
Comment (murgeyseb) [reply pages/documentation/simulation/flowdecomposition/index.md:35]: No, it is the impact of an injection *on* branch flow (or i miss something ?). I.E. sensitivity matrix

File: pages/documentation/simulation/flowdecomposition/index.md (hunk 5 above)
Comment (murgeyseb) [reply pages/documentation/simulation/flowdecomposition/index.md:75]: My spell checker does not agree with you on that but I can change it if you want

Comment (annetill) [reply pages/documentation/simulation/flowdecomposition/index.md:75]: Indeed it is just to be coherent with the rest of the site, I am not so good in English you know that!

Comment (murgeyseb) [reply pages/documentation/simulation/flowdecomposition/index.md]: For consistency with the rest of the documentation I put setpoint

**Review by annetill — APPROVED** [review]
Thanks a lot!

**Pull request merged**